### ISSUE TITLE
Add polyfill for Chrome

### ISSUE THREAD

**Issue by jure965** [jure965/OpenLorem#25]
Include polyfill for chrome and put standard web extension api where applicable.

**Comment by lyu4321** [comment]
Hey, I'm interested in this issue as well but I'm a bit new to polyfills. Would a library like this be suitable for this issue? https://github.com/mozilla/webextension-polyfill

**Comment by jure965** [comment]
Yes, I was also looking at this library. Might as well add it as a npm dependency.
